README.md

```diff
@@ -1,6 +1,6 @@
 <img src="images/razee_icon.png">
 
-[Razee](http://www.razee.io/) is an open-source project that was developed by IBM to automate and manage the deployment of Kubernetes resources across clusters, environments, and cloud providers, and to visualize deployment information for your resources so that you can monitor the rollout process and find deployment issues more quickly.
+[Razee](http://www.razee.io/) is an open-source project that was developed by IBM to automate and manage the deployment of Kubernetes resources across clusters, environments, and cloud providers, and to visualize deployment information for your aggregate resources so that you can monitor the rollout process and find possible deployment issues.
```

Comment:
```suggestion
[Razee](http://www.razee.io/) is an open-source project that was developed by IBM to automate and manage the deployment of Kubernetes resources across clusters, environments, and cloud providers, and to visualize deployment information for your multi-cluster resources so that you can monitor the rollout process and find potential deployment issues.
```